doc/nrf/protocols/matter/getting_started/adding_clusters.rst

```diff
@@ -317,7 +317,7 @@ To implement the callback function, complete the following steps:
 1. Create a new file, for example :file:`src/zcl_callbacks.cpp`.
 2. Implement the callback in this file:
 
-   a. Open :file:`src/zap-generated/callback-stub.cpp` to check the list of customizable callback functions, marked with ``__attribute__((weak))``.
+   a. Open :file:`ncs/modules/lib/matter/src/app/util/generic-callback-stubs.cpp` to check the list of customizable callback functions, marked with ``__attribute__((weak))``.
```

Comment:
Could you fix also the next step? The method description is in the `generic-callback.h` file not in the cpp, so it would be good to mention it.